ingestion-beam/bin/download-geolite2

```diff
@@ -6,13 +6,27 @@
 # from MaxMind for use in development.
 BASE_DIR="$(dirname "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )")"
 DOWNLOAD_URL="https://download.maxmind.com/app/geoip_download?edition_id=GeoLite2-City&license_key=${MM_LICENSE_KEY}&suffix=tar.gz"
```

Comment:
I wonder if it would make sense to have a check for whether `MM_LICENSE_KEY` is set in the script. Right now, even if it's not set it'll still try to download the files without erroring out. 

Having a error message printed out might make it easier for people setting up the project for the first time to figure out what's going wrong. Though there is documentation somewhere for this, so probably not super urgent.